src/tasks/submit-catalog/submit-catalog-v2/task.json

```diff
@@ -71,18 +71,43 @@
       "required": true,
       "helpMarkDown": "Specify the path and file name of the catalog submission file."
     },
+    {
```

Comment:
Can we detect type of file and have just one argument or are they both zip files?

Reply:
both of them are zip files